operations/iuf/workflows/product_delivery.md

```diff
@@ -52,7 +52,13 @@ Once this step has completed:
 
 - The `customizations.yaml` file has been updated per product documentation.
 
-## 3. Execute the IUF `deliver-product` stage
+## 3. Populate admin directory with files defining site preferences
+
+**`NOTE`** This subsection is optional and only required when installing USS 1.1 or higher.
```

Comment:
Why is this only necessary when installing USS 1.1 or higher? The next step refers to `site_vars.yaml` so the file must exist by the time the next step is run.

Reply:
Removed the Note . Added a new page referenced to it . Removed it from Configuration.md

Reply:
Removed reference to Configuration.md from Upgrade only CSM page as required content is removed from this page

Reply:
This is needed for USS-1.1 and higher because the PBS Pro and SLURM packages are now delivered with USS, so the USS team is using a product_var / site_var variable to enable to disable the use of particular WLMs.